src/test/java/com/artipie/rpm/hm/NodeHasPkgCount.java

```diff
@@ -51,6 +53,20 @@ public NodeHasPkgCount(final int count, final String tag) {
         );
     }
 
+    /**
+     * Constructor with {@link Description}.
+     * @param count Expected packages count.
+     * @param tag Xml tag.
+     * @param description Hamcrest description for matcher.
+     */
+    public NodeHasPkgCount(final int count, final String tag, final Description description) {
```

Comment:
@paulodamaso make this ctor protected because `description` is an internal mechanism which doesn't need to be injected other than for testing the matcher itself.